examples/ansible/setup_libvirt_host.yml

```diff
@@ -4,10 +4,14 @@
 # A example playbook showing how the setup an ALP system as a libvirt host.
 #
 
+# use ansible-playbook setup_libvirt_host.yml -e "playbook_path=$(pwd)" -e "ansible_user=$(whoami)" -e "reboot_mode=auto" to enable auto reboot while running as root ensure ansible container is installed as root.
```

Comment:
This comment is very long and should be reformatted as a multi-line comment:

```suggestion
# Use:
#   ansible-playbook setup_libvirt_host.yml -e "playbook_path=$(pwd)"
#       -e "ansible_user=$(whoami)"
#       -e "reboot_mode=auto"
# to enable auto reboot while running as root to ensure ansible container
# is installed as root.
```

Reply:
Also it would probably be better to say that the Ansible container needs to be pre-loaded into the target user's image cache to avoid long delays running ansible commands as that user, if it is not the same user that is running the current playbook.